.rubocop_todo.yml

```diff
@@ -0,0 +1,131 @@
+
+# Offense count: 1
+# This cop supports unsafe autocorrection (--autocorrect-all).
+Lint/BooleanSymbol:
+  Exclude:
+    - 'lib/puppet/provider/reboot/posix.rb'
+
+# Offense count: 2
+# Configuration parameters: AllowedMethods, AllowedPatterns, CountRepeatedAttributes.
+Metrics/AbcSize:
+  Max: 29
+
+# Offense count: 3
+# Configuration parameters: CountComments, CountAsOne, AllowedMethods, AllowedPatterns, inherit_mode.
+# AllowedMethods: refine
+Metrics/BlockLength:
+  Max: 192
+
+# Offense count: 10
+# Configuration parameters: CountComments, CountAsOne, AllowedMethods, AllowedPatterns.
+Metrics/MethodLength:
+  Max: 20
+
+# Offense count: 1
+# Configuration parameters: ForbiddenDelimiters.
+# ForbiddenDelimiters: (?i-mx:(^|\s)(EO[A-Z]{1}|END)(\s|$))
+Naming/HeredocDelimiterNaming:
+  Exclude:
+    - 'lib/puppet/type/reboot.rb'
+
+# Offense count: 1
+# Configuration parameters: MinNameLength, AllowNamesEndingInNumbers, AllowedNames, ForbiddenNames.
+# AllowedNames: as, at, by, cc, db, id, if, in, io, ip, of, on, os, pp, to
+Naming/MethodParameterName:
+  Exclude:
+    - 'lib/puppet/type/reboot.rb'
+
+# Offense count: 30
+# Configuration parameters: Prefixes, AllowedPatterns.
+# Prefixes: when, with, without
+RSpec/ContextWording:
+  Exclude:
+    - 'spec/acceptance/reboot_pending_spec.rb'
+    - 'spec/unit/provider/reboot/linux_spec.rb'
+    - 'spec/unit/provider/reboot/posix_spec.rb'
+    - 'spec/unit/provider/reboot/windows_spec.rb'
+    - 'spec/unit/task/init_spec.rb'
+    - 'spec/unit/type/reboot_spec.rb'
+
+# Offense count: 13
+# Configuration parameters: IgnoredMetadata.
+RSpec/DescribeClass:
+  Exclude:
+    - '**/spec/features/**/*'
+    - '**/spec/requests/**/*'
+    - '**/spec/routing/**/*'
+    - '**/spec/system/**/*'
+    - '**/spec/views/**/*'
+    - 'spec/acceptance/reboot_finished_spec.rb'
+    - 'spec/acceptance/reboot_immediate_spec.rb'
+    - 'spec/acceptance/reboot_message.spec.rb'
+    - 'spec/acceptance/reboot_no_refresh_spec.rb'
+    - 'spec/acceptance/reboot_pending_spec.rb'
+    - 'spec/acceptance/reboot_resume_spec.rb'
+    - 'spec/acceptance/reboot_timeout_spec.rb'
+    - 'spec/acceptance/tasks/init_spec.rb'
+    - 'spec/plans/init_spec.rb'
+    - 'spec/unit/provider/reboot/linux_spec.rb'
+    - 'spec/unit/provider/reboot/posix_spec.rb'
+    - 'spec/unit/provider/reboot/windows_spec.rb'
+    - 'spec/unit/type/reboot_spec.rb'
+
+# Offense count: 13
+# Configuration parameters: CountAsOne.
+RSpec/ExampleLength:
+  Max: 12
+
+# Offense count: 2
+RSpec/ExpectInHook:
+  Exclude:
+    - 'spec/unit/provider/reboot/linux_spec.rb'
+    - 'spec/unit/provider/reboot/posix_spec.rb'
+
+# Offense count: 1
+# Configuration parameters: Include, CustomTransform, IgnoreMethods, SpecSuffixOnly.
+# Include: **/*_spec*rb*, **/spec/**/*
+RSpec/FilePath:
+  Exclude:
+    - 'spec/acceptance/reboot_message.spec.rb'
+
+# Offense count: 22
+RSpec/MultipleExpectations:
+  Max: 3
```

Comment:
we can add this one to the `.rubocop.yml`. 
It's one of the agreed cops in the doc :-)

Reply:
addressed